### ISSUE TITLE
Tune hyperparameters

### ISSUE THREAD

**Issue by heitorc62** [codesnifferai/ai#15]
Hyperparamenters that we may pay attention:

- Sequence length
- Learning rate
- Train percent
